.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: node_js
+node_js:
+    - 8
+script:
+    - npm test
+    - npm run check
```

Comment:
I'd prefer to call the commands in reverse order, check first, test after that. Or we could run eslint in different job entirely. But I believe, in this case, reversing the order of commands should be enough.

You are also missing a newline at end of this file (as well as some others, as I pointed out before).

Reply:
I changed order of commands. I hope that I also added new line at the end of all files :)